main.go

```diff
@@ -20,10 +20,10 @@ const (
 	BUS_ERR
 )
 
-var configFile = flag.String("file", "", "Path to configuration file")
+var configFile = flag.StringP("config", "c", "", "Path to configuration file")
```

Comment:
Can we keep compatibility and deprecate out `--file` with a warning instead of a breaking change?